DOCSP-18158 updated change stream example (#256)
mongodb/docs-node: source/usage-examples/changeStream.txt

First, open the change stream on the collection and then define a callback
on the change stream using the ``on()`` method. Once set, generate a change

> event to emit by performing a change to the collection.

Reviewer: ```suggestion
event by performing a change to the collection.
```

Suggestion: Im not sure "to emit" is needed here, and it sounds odd that the change event was generated be "emitted", but rathe it was just generated.

To generate the change event on the collection, let's use ``insertOne()``
method to add a new document. Since the ``insertOne()`` may run before the
listener function can register, we use a timer, declared with

> ``simulateAsyncPause`` to wait 100 milliseconds before executing the insert.

Reviewer: ```suggestion
``simulateAsyncPause`` to wait 1 second before executing the insert.
```
Suggestion: I think seconds is clearer than milliseconds (I don't intuitively know how long 100 milliseconds is). 1 second only works though if the time interval is raised.

> listener function can register, we use a timer, declared with
> ``simulateAsyncPause`` to wait 100 milliseconds before executing the insert.

Reviewer: ```suggestion
listener function can register, we use a timer to wait 100 milliseconds before executing the insert.
```
Suggestion: I'm not sure "declared with" is right here. I think you could say "defined as", or I think it could just be called a timer.

We also use ``simulateAsyncPause`` after the insertion of the document
to provide ample time for the listener function to receive the change

> event and the for the callback to complete its execution before 

Reviewer: ```suggestion
event and for the callback to complete its execution before 
```
